src/Sulu/Bundle/MediaBundle/Resources/public/js/main.js

```diff
@@ -12,6 +12,9 @@ require.config({
         sulumedia: '../../sulumedia/js',
         sulumediacss: '../../sulumedia/css',
 
+        // if content will be registered later
+        sulucontentcss: '../../sulucontent/css',
```

Comment:
I don't get it... Would it work if you just remove it? If possible I would like to minimize the dependencies between the different bundles...